packages/zoe/src/contractFacet/zcfSeat.js

```diff
@@ -114,7 +114,9 @@ export const createSeatManager = (
   };
 
   const clear = zcfSeat => {
-    zcfSeatToStagedAllocations.delete(zcfSeat);
+    if (zcfSeatToStagedAllocations.has(zcfSeat)) {
```

Comment:
this is a change to the throw behavior of `clear`. was the earlier behavior documented or tested?

I'll go google it since [our API docs aren't on the APIs](https://github.com/Agoric/agoric-sdk/issues/4291) ;-) https://docs.agoric.com/zoe/api/zoe-contract-facet.html#zcfseat-clear Nope.

Reply:
Not only are there no tests for `clear`, there appear to be [no tests for zcfSeat](https://agoric-sdk-coverage.netlify.app/#file/desc/false/true/true/true/zoe%2Fsrc%2FcontractFacet/).  

[not suggesting this PR add anything.]